src/System.Linq.Expressions/src/System/Runtime/CompilerServices/RuntimeOps.MergedRuntimeVariables.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    public partial class RuntimeOps
```

Comment:
I think it is ok for this class to show up as public and be empty. These methods are not intended to be a public API and should not be a part of any kind of contracts. The [Obsolete] is a good indication of that.

I think that is an option # 1 - ok to make it public, but do not expose through facades. Ok to not have compiler-specific [Obsolete] members when compiling is not supported.